tas-initializer/Dockerfile.tas-initializer

```diff
@@ -0,0 +1,25 @@
+FROM registry.redhat.io/rhel9/go-toolset@sha256:f969e6a4fe53663f3fc8d0ddc17ec2fc3def2af8e68ce7061f016a598798f474 AS build-env
+
+WORKDIR /opt/app-root/src
+
+RUN curl -sL https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz -o /opt/app-root/src/openshift-client-linux.tar.gz && \
```

Comment:
Surely there must be a way to install `oc` that doesn't require `curl`. Maybe there is a base image we could use?

Reply:
I have not seen a single image in the certified Red Hat catalog that has the `kubectl` binary installed properly. What I do in the recent commit is that I override the add the yum repo for `kubectl` and install it there as discussed in the official `kubectl` documentation. Only issue is this is locked to an `x.y` release (`1.28`) so we may get `z` updates which break things. Pretty unlikely but possible.